### PR TITLE
made a sticky note with description of border

### PR DESCRIPTION
**Pull request by Wilko2001**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,7 +18,14 @@ <h2>Selectors</h2>
...
         <section class="dashboard--properties">
           <h2>Properties</h2>
-          <ul></ul>
+          <ul>
+            <div class="post-it" id="Emma-Wilkinson">
+            <h3>
+              <li>Border
+            </h3>
+            <p>A border is an outline box that goes around a property you assign it to.</p></li>
+            </div>
```
Comment (PerryBaran) [inline index.html:27]: should be wrapped in li tag

Comment (PerryBaran) [reply index.html:27]: the id should go inside of this li

File: styles.css [hunk 2]
```diff
@@ -87,7 +87,18 @@
...
 /***************************************************/
 /* Put Property post-it styles here! ↓ */
+.post-it {
+
+  aspect-ratio: 1:1;
+  height: 200px;
+  background-color: #FF6F91;
+  transform: rotate(1deg);
+  width: 250px;
+}
```
Comment (PerryBaran) [inline styles.css:97]: should make changes to the id with your name instead of the post-it class

File: index.html [hunk 3]
```diff
@@ -18,7 +18,14 @@ <h2>Selectors</h2>
 
         <section class="dashboard--properties">
           <h2>Properties</h2>
-          <ul></ul>
+          <ul>
+            <div class="post-it" id="Emma-Wilkinson">
```
Comment (PerryBaran) [inline index.html:22]: this should be the li element

**Review by PerryBaran — CHANGES_REQUESTED** [review]

**Review by PerryBaran — APPROVED** [review]
looks good to me

**Review by GarethBeer — APPROVED** [review]
You've used the correct li element and it looks good

**Pull request merged**